### PR TITLE
Feature-IOPS-2673-Mapping-of-ValueSet-to-Profiles

### PR DESCRIPTION
**Pull request by KazeemHamzat**

### CODE REVIEW

File: StructureDefinition/NHSDigital-ServiceRequest-Genomics.StructureDefinition.json [hunk 1]
```diff
@@ -174,6 +164,14 @@
         "definition": "A code that identifies a particular service (i.e., procedure, diagnostic investigation, or panel of investigations) that have been requested.\r\n\r\nFor Genomic Test Orders, use the system 'https://www.england.nhs.uk/publication/national-genomic-test-directories' and code from one of the test directories provided by NHS England. Addition of these codes to an appropriate coding system, e.g. SNOMED-CT, is under discussion.",
         "min": 1
       },
+      {
+        "id": "ServiceRequest.orderDetail",
```
Comment (omarisgreat) [inline StructureDefinition/NHSDigital-ServiceRequest-Genomics.StructureDefinition.json:168]: While you are profiling this, could you add another slice for the CodeSystem URL defined in NamingSystem-England-GenomicTestDirectory (to allow Test Directory Codes to also be used here without warnings)

**Comment by omarisgreat** [comment]
FYI, you will also need to update ServiceRequest and Bundle examples that contain reasonCode/orderDetail to use codes from the bound valuesets

File: StructureDefinition/NHSDigital-ServiceRequest-Genomics.StructureDefinition.json [hunk 2]
```diff
@@ -15,52 +35,18 @@
...
-            }
-          ],
-          "rules": "open"
-        },
         "min": 1
       },
-      {
-        "id": "ServiceRequest.extension:additionalContact",
-        "path": "ServiceRequest.extension",
-        "sliceName": "additionalContact",
```
[the remark this answers is not in the document]
Comment (KazeemHamzat) [reply StructureDefinition/NHSDigital-ServiceRequest-Genomics.StructureDefinition.json:32]: This extension is inherited from the base, thus, Forge is right to remove these extra codes from NHSDigital_ServiceRequest_Genomics.

**Comment by KazeemHamzat** [comment]
> FYI, you will also need to update ServiceRequest and Bundle examples that contain reasonCode/orderDetail to use codes from the bound valuesets

Update ServiceRequest and Bundle examples will be done in a new branch.

**Review by omarisgreat — APPROVED** [review]
Looks good to me

**Pull request merged**